dataStorage/storage.js

```diff
@@ -31,14 +31,14 @@ async function loadAsync(model, id, update) {
     let data = await loadFromOffilneStorageAsync(model.key, id);
     if (!data || data.length == 0) {
         // TODO need to sync from server;
-        data = await loadFromCloudAsync(model, id);
+        data = await loadFromCloudAsync(model, id, false);
         data && saveToOffilneStorageAsync(data, model.key, id);
```

Comment:
nit: would it better to use the traditional coding style?
if (data) {
    saveToOffilneStorageAsync(data, model.key, id); 
}